src/main/java/nonapi/io/github/classgraph/fastzipfilereader/LogicalZipFile.java

```diff
@@ -609,11 +613,51 @@ private void readCentralDirectory(final ZipFileSliceReader zipFileSliceReader, f
                 }
                 final boolean isDeflated = compressionMethod == /* deflated */ 8;
 
+                final int lastModifiedTime = entryBytes != null ? ZipFileSliceReader.getShort(entryBytes, entOff + 12)
+                        : zipFileSliceReader.getShort(cenPos + entOff + 12);
+
+                final int lastModifiedDate = entryBytes != null ? ZipFileSliceReader.getShort(entryBytes, entOff + 14)
+                        : zipFileSliceReader.getShort(cenPos + entOff + 14);
+
+                // MS-DOS Date & Time Format
+                final int lastModifiedSecond = (lastModifiedTime & 0b11111) * 2;
+                final int lastModifiedMinute = lastModifiedTime >> 5 & 0b111111;
+                final int lastModifiedHour = lastModifiedTime >> 11;
+                final int lastModifiedDay = lastModifiedDate & 0b11111;
+                final int lastModifiedMonth = (lastModifiedDate >> 5 & 0b111) - 1;
+                final int lastModifiedYear = (lastModifiedDate >> 9) + 1980;
+
+                final Calendar lastModifiedCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
```

Comment:
Zipfiles don't have a timezone recorded anywhere in them -- so I guess it is it fair to pick UTC... Do you know what different zip tools do with this? Is UTC an ad hoc standard for zip entry timestamps?

Reply:
AFAICT the only viable option is UTC but details are hard to find.

Reply:
UTC is the only reasonable default, so this is fine. I updated the JavaDoc (after pushing out the release though) to point out this possibly erroneous default assumption.